docs/src/man/basics.md

```diff
@@ -2068,3 +2132,212 @@ options of the transformation mini-language. More advanced examples, in particul
 showing how to pass or produce multiple columns using the `AsTable` operation
 (which you might have seen in some DataFrames.jl demos) are given in the later
 sections of the manual.
+
+## Broadcasting with Transformation Functions
+
+[Broadcasting](https://docs.julialang.org/en/v1/manual/arrays/#Broadcasting)
+is a convenient way to apply multiple transformations
+within one call to a transformation function.
+Broadcasting within the `Pair` of a `transformation` is no different than
+broadcasting in base Julia.
+The broadcasting `.=>` will be expanded into a vector of pairs,
+and this expansion will occur before the tranformation function is invoked.
```

Comment:
maybe give some example here before moving to explanation of technical aspects what happens underneath?

Reply:
I didn't touch the Examples section between the two sections I wrote since it covered most things. It would probably be better to rewrite and split it up, but that might be a separate PR.

Reply:
But the question is (maybe you are right though) if people will be able to digest the rules without seeing the examples?